### ISSUE TITLE
issue with port forwarding

### ISSUE THREAD

**Issue by haritha-j** [tananaev/traccar-client-android#234]
Hi, im a bit new to this, so forgive my ignorance. So once i set this up to the best of my ability, i can get this to detect my phone when it's connected to the same wifi network as the laptop im running the server on. But over the internet (when my phone is running on mobile data) my phone's location isnt picked up. I've forwarded the port 5055 to my laptop's static ip address, yet your port checker says port 5055 is closed.
I set my laptop's public ip address as the server address on my phone's client, i hope that was what i was supposed to do.

Also on an unrelated note, how could i customize the map shown by say highlighting a route on the map?

**Comment by tananaev** [comment]
Are you sure your ISP provides you with dedicated public IP address? If yes, the problem must be either in router or firewall configuration.

**Comment by tananaev** [comment]
By customizing the map you mean adding arbitrary objects on it? It's not possible at the moment.

**Comment by haritha-j** [comment]
Yeah, I turned the firewall off entirely as well so I'm guessing it's a problem with my router. Do you have any suggestions on what night be the issue and how to fix it?

**Comment by haritha-j** [comment]
And yes I mean adding arbitrary objects, specifically I want to highlight bus routes on the map. Is there any workaround I can use, for instance use an alternative to openmaps that is customizable by the user. Thanks for your help.

**Comment by tananaev** [comment]
> Do you have any suggestions on what night be the issue and how to fix it?

No, I have no idea. It can be anything. Are you positive that you get a dedicated IP address from provider?

> Is there any workaround I can use, for instance use an alternative to openmaps that is customizable by the user.

Yes, you can use custom maps:

https://www.traccar.org/map-layers/

**Comment by haritha-j** [comment]
Thanks for the feedback. I've always assumed that I have a dedicated ip. But yeah I'm not sure. Is there any way of checking?

**Comment by tananaev** [comment]
Contact your ISP. I don't know any other reliable way.

**Comment by haritha-j** [comment]
yeah, i will do that. thanks for your help.
